src/bot/commands/approve.ts

```diff
@@ -298,6 +318,8 @@ export default class ApproveCommand extends Command {
     try {
       if (isSuggestion) {
```

Comment:
Ici l'opterait pour un switch :
```js
switch (channel.id) {
	case suggestionsChannelId: {
	  await this.approveSuggestion(interaction, proposal, message, embed);
	  break;
	}
	case reportsChannelId: {
	  await this.approveReport(interaction, proposal, message, embed);
	  break;
	}
	...
}
```

Reply:
Pas bête ! Je me dis aussi, est-ce que ce serait possible de faire un `this['approve${<...>}`]` avec le ProposalType ?

Reply:
Non, c'est pas considéré comme propre.

Reply:
D'accodac